### ISSUE TITLE
[Bug] latest git build on arch crashes

### ISSUE THREAD

**Issue by mxaddict** [mdgaziur/findex#70]
## Description
Latest git build on arch (using aur findex-git package) crashes when launched

## To Reproduce
Just press the configured hotkey (on my system it's wayland so hot key just touches a file on disk)

## Expected behavior
Should launch findex

## System information
 - Distro: arch
 - Findex Version: findex-git-0.8.2.12-1

## Output of `rustc --version --verbose` (if you compiled findex on your machine):
```
rustc 1.82.0 (f6e511eec 2024-10-15)
binary: rustc
commit-hash: f6e511eec7342f59a25f7c0534f1dbea00d01b14
commit-date: 2024-10-15
host: x86_64-unknown-linux-gnu
release: 1.82.0
LLVM version: 19.1.1
```

## Log/Output (if applicable)
```
[INFO] Starting Findex...
[INFO] listening for hotkey...
thread 'main' panicked at crates/findex/src/gui/mod.rs:222:39:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'main' panicked at core/src/panicking.rs:221:5:
panic in a function that cannot unwind
```

**Comment by Longestboi** [comment]
Oof, I recreated this issue in an Arch container (using distrobox) and I think I fixed it with pull request #71.

Do you mind testing if the pr fixed the issue @mxaddict?

**Comment by mdgaziur** [comment]
Closing as it's supposedly been fixed in #71.
